pfs/lineLists/skyLines.txt

```diff
@@ -535,19 +535,20 @@
 681.84080            0.00    OH           1 UNKNOWN   8
 681.94760            0.00    OH           1 UNKNOWN   8
 682.93300            0.58    OH         256 7-2_R1f(3.5)  12
-682.93430            0.58    OH         512 7-2_R1f(3.5)|7-2_R1e(3.5)  12
+682.93430            0.58    OH         768 7-2_R1f(3.5)|7-2_R1e(3.5)  12
```

Comment:
`768 = 0x300 = MERGED|COMBINED`. Why are lines that were `COMBINED` now being also flagged as `MERGED`???